File: fsharp-functions/OpenAPI/src/model/RequestTimeFilterFast.fs
namespace OpenAPI.Model

open System
open System.Collections.Generic
open Newtonsoft.Json
open OpenAPI.Model.RequestLocation
open OpenAPI.Model.RequestTimeFilterFastArrivalSearches

module RequestTimeFilterFast = 

  //#region RequestTimeFilterFast

  [<CLIMutable>]
  type RequestTimeFilterFast = {
    [<JsonProperty(PropertyName = "locations")>]
    Locations : RequestLocation[];
    [<JsonProperty(PropertyName = "arrival_searches")>]
    ArrivalSearches : RequestTimeFilterFastArrivalSearches;
  }
  
  //#endregion
  

File: csharp-dotnet2/src/main/CsharpDotNet2/Org/OpenAPITools/Model/RequestTimeFilterFast.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace Org.OpenAPITools.Model {

  /// <summary>
  /// 
  /// </summary>
  [DataContract]
  public class RequestTimeFilterFast {
    /// <summary>
    /// Gets or Sets Locations
    /// </summary>
    [DataMember(Name="locations", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "locations")]
    public List<RequestLocation> Locations { get; set; }

    /// <summary>
    /// Gets or Sets ArrivalSearches
    /// </summary>
    [DataMember(Name="arrival_searches", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "arrival_searches")]
    public RequestTimeFilterFastArrivalSearches ArrivalSearches { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class RequestTimeFilterFast {\n");
      sb.Append("  Locations: ").Append(Locations).Append("\n");
      sb.Append("  ArrivalSearches: ").Append(ArrivalSearches).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}
